roles/openshift_common/vars/infra_vars.yml

```diff
@@ -0,0 +1,5 @@
+role: infra
```

Comment:
I would suggest to prefix all these variables with something like add_machineset_, for example:
```
add_machineset_role
add_machineset_ami_id
add_machineset_instance_type
```
While verbose, it makes it clear on the caller side that these variables belong to this role. Also, it is easier to grep for these variable in the entire code base as their name is unique.

Reply:
done